doc/modules/language-guide/pages/motoko-introduction.adoc

```diff
@@ -4,35 +4,42 @@
 :sdk-short-name: DFINITY Canister SDK
 :sdk-long-name: DFINITY Canister Software Development Kit (SDK)
 
-{proglang} is a modern programming language you can use specifically for link:../developers-guide/concepts/what-is-ic{outfilesuffix}#ic-overview[{IC} platform] programs or as a general-purpose language for programs running on other platforms.
+{proglang} is a modern, general-purpose programming language you can use specifically to author link:../developers-guide/concepts/what-is-ic{outfilesuffix}#ic-overview[Internet Computer]  canister smart contracts.
```

Comment:
s/author link/link ?

Reply:
Not sure what you wanted here? Maybe make it as a suggestion? @osa1

Reply:
Sorry, this looks good, I just misread it. Closing.